libenkf/src/site_config.c

```diff
@@ -577,9 +577,9 @@ static bool site_config_init(site_config_type * site_config, const config_conten
 
   if (config_content_has_item(config, UMASK_KEY)) {
     const char * string_mask = config_content_get_value(config, UMASK_KEY);
-    mode_t umask_value;
+    int umask_value;
     if (util_sscanf_octal_int(string_mask, &umask_value))
-      site_config_set_umask(site_config, umask_value);
+      site_config_set_umask(site_config, (mode_t)umask_value);
```

Comment:
I would have preferred `util_sscanf_octal_int` to be refactored into `util_sscanf_octal_mode_t`. This is the only place this function is used within `libecl` and `libres`..

Reply:
I agree that this should have been taken out of `ecl` and made into a `..._mode_t`.